css/styles.css

```diff
@@ -321,3 +402,290 @@ textarea {
   left: 4px;
   font-size: 11px;
 }
+
+.desktopnav {
+  visibility: hidden;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: lightgray;
+}
+
+.desktopnav ul li {
+  margin-right: 30px;
+}
+
+.awesomedeco1 {
+  background-image: url(../resources/image-geometry_7.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 1488px;
+  top: 296px;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+}
+
+.awesomedeco2 {
+  background-image: url(../resources/image-geometry_6.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+  left: -86px;
+}
+
+.imggeo2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imggeo5 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.fixGap1 {
+  margin-bottom: -5px;
+}
+
+.imgGroup83 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.redSquare {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imageDots2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+@media screen and (min-width: 1680px) {
+  .decoration1 {
+    visibility: hidden;
+  }
+
+  .decoration2 {
+    visibility: hidden;
+  }
+
+  .decoration3 {
+    background-image: url(../resources/Thur.png);
+    background-repeat: no-repeat;
+    padding-bottom: 164px;
+    height: 877.09px;
+    visibility: visible;
+    width: 1403px;
+    margin-left: 677px;
+    position: relative;
+  }
+
+  .desktopflex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .desktopnav {
+    visibility: visible;
+  }
+
+  .dflex2 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+  }
+
+  .grid1 {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+  }
+
+  .header1 {
+    margin-top: -94px;
+  }
+
+  .rectangle1 {
+    margin-bottom: 0;
+  }
+
+  .rectangle2 {
+    margin-left: 0;
+    margin-bottom: 12px;
+  }
+
+  .weirddeco1 {
+    position: relative;
+    visibility: visible;
+  }
+
+  .weirddeco2 {
+    position: absolute;
+    left: 34px;
+    top: 113px;
+  }
+
+  .weirddeco3 {
+    position: absolute;
+    left: 207px;
+    top: 51px;
+  }
+
+  .dottedimg1 {
+    visibility: visible;
+    position: absolute;
+    height: 175px;
+    width: 175px;
+    top: 701px;
+    left: -174px;
+  }
+
+  .awesomedeco1 {
+    visibility: visible;
+  }
+
+  .awesomedeco2 {
+    visibility: visible;
+  }
+
+  .getstarted {
+    width: 100%;
+    margin-left: 0;
+  }
+
+  .projectsrotate {
+    font-size: 110px;
+    transform: rotate(-90deg);
+  }
+
+  .box3 {
+    height: 0;
+    width: 0;
+    visibility: hidden;
+  }
+
+  .cooldeco1 {
+    visibility: visible;
+  }
+
+  .box1 {
+    position: absolute;
+    left: 880px;
+    top: -51px;
+    width: 960px;
+  }
```

Comment:
- [ ] Kindly edit this section (the text, the badges, and the button) to make it match Figma

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187050305-cf40dea1-0482-43e1-90e6-5543674ab0f6.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187050311-e4675ff4-6bce-470c-aeb0-412817f6534f.png)